### ISSUE TITLE
Add "F12" Hot key to édit name of part in part panel

### ISSUE THREAD

**Issue by Gai-Luron** [musescore/MuseScore#20855]
### Your idea

Add "F12" Hot key to édit name of part in part panel. I push 
I systematically press this button out of habit.

### Problem to be solved

Add "F12" Hot key to édit name of part in part panel.
This is a standard shorcut in windows. Maybe you don"t have add this feature due to multiplatform support

**Comment by bkunda** [comment]
Are you sure it's F12 that's standard in Windows? (Not F2)?

**Comment by Gai-Luron** [comment]
Envoyé depuis mon téléphone-------- Message original --------De : bkunda ***@***.***>Date : mar. 9 janv. 2024 à 15:46À : musescore/MuseScore ***@***.***>Cc : Robert BRACCAGNI ***@***.***>, Author ***@***.***>Objet : Re: [musescore/MuseScore] Add "F12" Hot key to edit name of part in part panel (Issue #20855)
Are you sure it's F12 that's standard in Windows? (Not F2)?

**Comment by Gai-Luron** [comment]
You are right. Sorry. Envoyé depuis mon téléphone-------- Message original --------De : bkunda ***@***.***>Date : mar. 9 janv. 2024 à 15:46À : musescore/MuseScore ***@***.***>Cc : Robert BRACCAGNI ***@***.***>, Author ***@***.***>Objet : Re: [musescore/MuseScore] Add "F12" Hot key to edit name of part in part panel (Issue #20855)
Are you sure it's F12 that's standard in Windows? (Not F2)?
